lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java

```diff
@@ -608,15 +608,56 @@ public void testAnnotationAvroIgnore() throws Exception {
         + "\"org.apache.avro.reflect.TestReflect\",\"fields\":[]}");
   }
 
+  @AvroMeta(key = "X", value = "Y")
```

Comment:
I'd recommend not changing RAvroMeta and testAnnotationAvroMeta, leave them as they are to verify that the single annotation is still working. RAvroMultiMeta and the associated test case already verifies repeated meta annotations. Otherwise looks good to me!

Reply:
So the reason I added to this test case is we did not previously assert that the class-level annotation was respected in any tests. That usage was supported before the enhancement because it was on different elements of the same clas. I can put this into its own test if you prefer though

Reply:
Okay, fair enough, thanks! I missed that the case with class level annotation is not tested.